app/routers/api/remote.py

```
@@ -862,8 +869,8 @@ async def receive_chunk(
                 disk_space_validator.validate_disk_space_direct(file_size, Path(base_dir))
                 logger.debug(f"Disk space validated for {file_size} bytes at {base_dir}")
             except ValueError as e:
-                logger.error(f"Insufficient disk space for job {headers.job_id}: {e}")
-                raise HTTPException(status_code=507, detail=str(e)) from None
+                logger.exception(f"Insufficient disk space for job {headers.job_id}")
```

Ruff (0.15.7)

[warning] Logging statement uses f-string

(G004)